### ISSUE TITLE
resetForm() = this.getInitialState is not a function

### ISSUE THREAD

**Issue by timothyallan** [tannerlinsley/react-form#65]
Hiya, I'm using React 15.5.4 and React Form 1.2.7, and using resetForm() is now giving me `this.getInitialState is not a function` errors?
Happens in this area here:
```
 key: 'resetForm',
    value: function resetForm() {
      return this.setFormState(this.getInitialState());
    }
```

**Comment by DjLeChuck** [comment]
You're right. It's caused by this commit: https://github.com/tannerlinsley/react-form/commit/9035b82234b862d96ee4c6550bcba9982e5ef907#diff-62acef2590804164cb77b4b5aced00ebL38

**Comment by timothyallan** [comment]
This looks like it's been fixed with #69 . Just waiting on a push to NPM with the update!

**Comment by tannerlinsley** [comment]
Pushed 1.2.8
